linux_os/guide/system/software/gnome/gnome_screen_locking/dconf_gnome_screensaver_mode_blank/rule.yml

```diff
@@ -1,10 +1,23 @@
 documentation_complete: true
 
-prodtype: fedora,ol7,ol8,rhel7,rhel8
+prodtype: fedora,ol7,ol8,rhel7,rhel8,sle12,sle15
 
 title: 'Implement Blank Screensaver'
 
 description: |-
+    {{# The SLE12 and SLE15 STIG unfortunately requires per-user settings, #}}
+    {{# not a globally locked one, so we don't require but only suggest the latter. #}}
+    {{% if product in ["sle12","sle15"]%}}
+    On SUSE users should set the screensaver to use publicly viewable images or blank screen
+    by doing the following to configure the system.  Find the
+    Settings menu and then navigate to the Background selection section:
+    <tt>- Click "Activities" on the top left.</tt>
```

Comment:
@yarunachalam I am sorry I should have caught this The directions are slightly different, so I think that this block needs to be:
   {{% if product == "sle15" }}
   <tt>- Click "Activities" on the top left.</tt>
   <tt>- Click "Show Applications" at the bottom of the Activities menu.</tt>
   <tt>- Click the "Settings" icon.</tt>
   <tt>- Click "Background" from left hand menu.</tt>
   <tt>- Select image and set the Lock Screen image to the user's choice.</tt>
   <tt>- Exit Settings Dialog.</tt>
   {{%- else %}}
   <tt>- - Click "Applications" on the bottom left.</tt>
   <tt>- - Hover over "System Tools" with the mouse.</tt>
   <tt>- - Click the "Settings" icon under System Tools.</tt>
   <tt>- - Click "Background" and then "Lock Screen".</tt>
   <tt>- Select image and set the Lock Screen image to the user's choice.</tt>
   <tt>- Click "Select".</tt>
   <tt>- Exit Settings Dialog.</tt>
   {{%- endif %}}

Reply:
Fixed the description text.